services/core-web/common/models/documents/document.ts

```diff
@@ -69,6 +71,7 @@ export class MineDocument {
     this.archived_by = jsonObject.archived_by;
     this.archived_date = jsonObject.archived_date;
     this.is_latest_version = jsonObject.is_latest_version ?? true;
+    this.entity_title = jsonObject.project_title;
```

Comment:
So- the entity title should be on the MineDocument, but it should be the Type that _has a project title_ that sets it as such. Maybe = jsonObject.entity_title ?? "" so it's still defined but properly scoped.

Reply:
As discussed, this has been adjusted now.